packages/terra-search-field/src/SearchField.jsx

```diff
@@ -89,6 +94,7 @@ const defaultProps = {
   searchDelay: 250,
   value: undefined,
   inputAttributes: undefined,
+  enableAutoFocus: true,
```

Comment:
Should this be `false` by default?

Reply:
Booleans should default to false whenever possible. Booleans that default to true introduce a discouraged API. 

```jsx
<SeachField autoFocus={false} />
```

Vs

Auto focus on:
```jsx
<SeachField autoFocus />
```
Auto focus off:
```jsx
<SeachField />
```